strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/PypiArtifactGenerator.java

```diff
@@ -179,11 +189,14 @@ private void createSourcePackageFiles(ZipOutputStream zos,
         byte[] sourcesContent = String.format(SOURCES_CONTENT, scriptPath, eggPkgInfoPath, sourcesPath,
                                               dependencyLinksPath, topLevelPath).getBytes();
```

Comment:
Cosmetic: (I know this is not part of your changes), but could you please chop these arguments as one per line like this:
```
        byte[] sourcesContent = String.format(SOURCES_CONTENT,
                                              scriptPath,
                                              eggPkgInfoPath,
                                              sourcesPath,
                                              dependencyLinksPath,
                                              topLevelPath).getBytes();
```